settings.gradle

```diff
@@ -25,3 +25,4 @@ include ':androidtest'
 include ':settings'
 include ':servicetest'
 include ':maps'
+include ':osmdroid'
```

Comment:
I would consider converting some of those classes you have moved to this new module to kotlin.